packages/visx-demo/src/sandboxes/visx-shape-splitlinepath/package.json

```diff
@@ -12,6 +12,7 @@
     "@visx/responsive": "latest",
     "@visx/scale": "latest",
     "@visx/shape": "latest",
+    "lodash": "^4.17.21",
```

Comment:
looks like we don't have `lodash` as a `@visx/demo` dep. It's a little annoying / fragile, but sandboxes need all of their deps so they work as an independent app, but adding them here doesn't add them to the demo package. so can we do the following?

- add them to the demo `package.json`
- run `yarn` in the root to update the `yarn.lock` file